huxley/www/static/js/huxley/stores/__tests__/AssignmentStore-test.js

```diff
@@ -24,6 +25,10 @@ describe('AssignmentStore', function() {
     });
   });
 
+  it('subscribes to the dispatcher', function() {
+    expect(Dispatcher.register).toBeCalled();
+  });
+
```

Comment:
Not sure how necessary this is. Added it in because we're suppose to overwrite the callback that gets passed into `Dispatcher.register`.